pages/chat/_slug.vue

```diff
@@ -18,7 +18,7 @@
           <div class="text-xs space-x-1 text-gray-900 flex">
             <TAvatar name :uid="item.createdBy" />
             <span>•</span>
-            <div>{{ dateDiff(item.createdAt) }} ago</div>
+            <div>{{ dateDiff(item.createdAt) }} {{ $t('date') }}</div>
```

Comment:
Please remove it. `dateDiff` function already includes `ago` word.
And also if you would use it `date` should belong to some group. It's too vague like that.

```suggestion
            <div>{{ dateDiff(item.createdAt) }}</div>
```